_docs_integrate/requests-via-relationshiptemplates.md

```diff
@@ -27,107 +27,141 @@ required_by:
+If a Request is contained within a RelationshipTemplate, its validity is automatically checked when the [RelationshipTemplate is created](#create-the-relationshiptemplate). The RelationshipTemplate cannot be created if the Request is faulty.
+Nevertheless, the Request's validity should be checked before attempting to create the RelationshipTemplate in order to obtain additional information about the reasons for the error in the case of a faulty Request.
 {: .notice--info}
 
 ## Create the RelationshipTemplate
 
-Next, [create a RelationshipTemplate]({% link _docs_use-cases/use-case-transport-create-own-relationshiptemplate.md %}) on the Sender Connector, using a [RelationshipTemplateContent]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplatecontent).
-If there is no Relationship between the Connectors, yet, you must put the `content` property of the payload of the previous step in the `onNewRelationship` property of the RelationshipTemplateContent.
-However, if the Connectors already have a Relationship with each other, you must put it in the `onExistingRelationship` property of the `RelationshipTemplateContent`.
-But note, that the `onNewRelationship` property is required and must, therefore, always be set.
+Next, the Sender wants to create the RelationshipTemplate, which contains the Request it wants to send to the Recipient. To specify a Request within a RelationshipTemplate, a data object of type [RelationshipTemplateContent]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplatecontent) must be used within the `content` property of the [RelationshipTemplate]({% link _docs_integrate/data-model-overview.md %}#relationshiptemplate).
+A Request can be specified in the `onNewRelationship` property or the `onExistingRelationship` property of the RelationshipTemplateContent.
+If no Relationship has yet been established between the Sender and the Recipient, the Recipient receives the Request specified in the `onNewRelationship` property when loading the RelationshipTemplate.
+However, if an active Relationship already exists, the Recipient receives the Request specified in the `onExistingRelationship` property when loading the RelationshipTemplate, if such is specified. The specification of a Request in the `onNewRelationship` property is mandatory in contrast to the specification of a Request in the `onExistingRelationship` property.
+
+Note that the same Request can be specified in the `onExistingRelationship` property as in the `onNewRelationship` property, but a different Request can also be used.
+These customization options are useful as the Recipient that will load the RelationshipTemplate may not be known in advance.
+It is therefore possible that a Relationship between the Sender and the Recipient already exists or not.
+It is also possible to configure the RelationshipTemplate so that it can be loaded by multiple Identities.
```

Comment:
Repetition of "It is". Since the sentences describe independent features, I'd vary their beginnings. E.g. you could start the second sentence with "Furthermore, it is possible to"